config/routes.rb

```diff
@@ -15,6 +15,7 @@
   get '/auth/google_oauth2/callback', to: 'sessions#google', as: 'auth_google'
 
   resources :users, only: [:show, :new, :create] do
```

Comment:
We need to figure out how to adapt this since we will not use users controller anymore.